### PR TITLE
Ruby on Rails capstone project - Budget app: Final Milestone

### PR DESCRIPTION
**Pull request by pasytchangwa**
In this milestone, the following was done:

- Added a Splash screen
- Added a Home page (categories page)
- Added a Transactions page
- Added a new category" page
- Added a new transaction" page
- Create unit and integration tests for all the most important components of your RoR application.

test run

- used Postgres as my database.
- Used device for authentication.
- Validated all user input to make sure that anyone with bad intentions cannot compromise your app.

- The project is deployed and accessible online.
[Live Demo](https://quiet-river-23201.herokuapp.com/)

- PROJECT VIDEO
https://www.loom.com/share/7f2e572ba7e749b28de48613db292b7c

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,111 @@
...
 
-* Configuration
+## Built With
 
-* Database creation
+- Ruby on Rails
+- PostgreSQL
 
-* Database initialization
+## Live Link
```
Comment (arslanbisharat) [inline README.md:16]: - [x] please add a video recording as requested in the project guidelines

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [x] Make sure you are following the correct Gitflow. 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (pasytchangwa) [reply README.md:16]: Video added to the readme but I already had those in my pull request

Comment (peterrobert) [reply README.md:16]: This change has been implemented well

**Review by peterrobert — APPROVED** [review]
Hi @pasytchangwa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**